### PR TITLE
fix(select): make @Output names consistent #6677

### PR DESCRIPTION
**Pull request by amcdnl**
Updated APIs per #6677 

- `onOpen` to `opened`
- `onClose` to `closed`
- `change` to `selectionChange`

### CODE REVIEW

File: src/lib/select/select.ts [hunk 1]
```diff
@@ -402,13 +402,31 @@ export class MdSelect extends _MdSelectMixinBase implements AfterContentInit, On
   }
 
   /** Event emitted when the select has been opened. */
-  @Output() onOpen: EventEmitter<void> = new EventEmitter<void>();
+  @Output() opened: EventEmitter<void> = new EventEmitter<void>();
```
Comment (jelbourn) [inline src/lib/select/select.ts]: Should we make this consistent with sidenav and just have one `openedChange`?
Thoughts @kara @crisbeto?

Comment (crisbeto) [reply src/lib/select/select.ts]: That's fine by me, we just have to make sure to mark it as a breaking change.

**Comment by amcdnl** [comment]
@jelbourn - let me know what you want to do regarding that.

**Comment by jelbourn** [comment]
If we're making the sidenav `openChange` then we should do that here too

**Comment by amcdnl** [comment]
@jelbourn - should be good to go now.

**Comment by jelbourn** [comment]
Caretaker notE: should be completely backwards compatible, so if something fails there's a bug

**Comment by amcdnl** [comment]
closing in favor of #8052

**Pull request closed without merging**